components/batch-system/src/test_runner.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+//! A sample Handler for test and micro-benchmark purpose.
+
+use crate::*;
+use std::borrow::Cow;
+use std::sync::{Arc, Mutex};
+use tikv_util::mpsc;
+
+pub enum Message {
+    Loop(usize),
+    Callback(Box<dyn FnOnce(&mut Runner) + Send + 'static>),
+}
+
+pub struct Runner {
```

Comment:
Could you add some docs for what this is doing and how? And how the various fields are used please?